src/basic/namespace-util.c

```diff
@@ -3,13 +3,18 @@
 #include <fcntl.h>
 #include <sys/ioctl.h>
 #include <sys/mount.h>
+#if WANT_LINUX_FS_H
+#include <linux/fs.h>
+#endif
```

Comment:
Please sort.

Reply:
I saw a comment in src/core/execute.c mentioning that <linux/fs.sh> must be included after <sys/mount.h> so that's why I kept it unsorted. I see the same in src/basic/mountpoint-util.c. It seems the order matters, could you confirm?

Reply:
yeah keep it as-is